packages/wallets/src/features/cashier/modules/Transactions/Transactions.scss

```diff
@@ -19,6 +19,16 @@
         }
     }
 
+    // don't need details/error message
+    .wallets-textfield__message-container {
+        display: none;
+    }
+
+    // don't show text selection
+    .wallets-textfield__field::selection {
+        background: none;
+    }
```

Comment:
This fixes this

https://github.com/binary-com/deriv-app/assets/119863957/385ecf40-14ce-4b64-838a-5e68c792d575